bin/compile

```diff
@@ -40,6 +40,29 @@ export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}"
 cd $BUILD_DIR
 dotnet --info
 
+function publish() {
+	echo "publish $1"
+	dotnet publish $1 --output $2 --configuration Release --runtime linux-x64
+}
+
+if [ -e "$BUILD_DIR/Procfile" ]; then
+    numberOfProjects=0
+	while read line || [[ -n "$line" ]];
```

Comment:
Hi @WilhelmKleuSage 

It does mean support of multiple projects? I think we can't do it, because only one web process instruction per file available

Reply:
Apologies for the delay in reply, I was away.
The code iterates the entries in the Procfile, try to match a valid dll and then publish.
Use the count of valid projects to determine exit code. 
While there can only be one `web`  process type there can be multiple other arbitrarily named processes (worker processes).